src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -128,26 +128,14 @@
          * @param req Original request
          * @throws IOException If fails
          * @checkstyle ExecutableStatementCountCheck (2 lines)
-         * @todo #558:30min Base ctor. According to new qulice version,
-         *  constructor must contain only variables initialization and other
-         *  constructor calls. Refactor code according to that rule and
-         *  remove `ConstructorOnlyInitializesOrCallOtherConstructors`
-         *  warning suppression.
          */
-        @SuppressWarnings
-            (
-                "PMD.ConstructorOnlyInitializesOrCallOtherConstructors"
-            )
         public Base(final Request req) throws IOException {
             super(req);
-            final InputStream stream = new RqLengthAware(req).body();
-            this.body = Channels.newChannel(stream);
-            this.buffer = ByteBuffer.allocate(
-                // @checkstyle MagicNumberCheck (1 line)
-                Math.min(8192, stream.available())
-            );
+            this.body = Base.body(req);
```

Comment:
@dalifreire making a static method is a workaround not a solution. Here you could store the stream as a field and create Channel in `requests` method and pass the created variable to `copy` and `make`.